### PR TITLE
Modify paysource

### PR DESCRIPTION
**Pull request by haritsfahreza**
Hi @rizdaprasetya and Midtrans team!

Based on Snap API official documents, I proposed to add `danamon_online` and remove `xl_tunai` and `bbm_money` on `paysource`. Please take a look. 

Thanks,
Harits Fahreza

### CODE REVIEW

File: paysource_test.go [hunk 1]
```diff
@@ -15,8 +15,6 @@ func TestPaymentType(t *testing.T) {
 	is.Equal("mandiri_clickpay", midtrans.SourceMandiriClickpay)
```
Comment (rizdaprasetya) [inline paysource_test.go]: @haritsfahreza great contribution 👍 
One step away from perfect,
How about adding `gopay` & `danamon_online` to the `paysource_test.go`?

Comment (haritsfahreza) [reply paysource_test.go]: Done @rizdaprasetya please take a look 🙏

**Comment by rizdaprasetya** [comment]
Perfect, I'll merge it now!

**Comment by rizdaprasetya** [comment]
Thanks for your continuous contribution 😃

**Pull request merged**